### PR TITLE
Update profile scores in line with the web design

### PR DESCRIPTION
**Pull request by EVAST9919**
Supersedes https://github.com/ppy/osu/pull/5442

Screenshot_1

### CODE REVIEW

File: osu.Game/Overlays/Profile/Sections/ProfileItemBackground.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        protected override IEnumerable<Drawable> EffectTargets => new[] { background };
+        protected override Container<Drawable> Content => content;
+
+        private readonly Box background;
+        private readonly Container content;
+
+        public ProfileItemBackground()
+        {
+            RelativeSizeAxes = Axes.Both;
+            Enabled.Value = true; //manually enabled, because we have no action
```
Comment (bdach) [inline osu.Game/Overlays/Profile/Sections/ProfileItemBackground.cs]: Not sure how I feel about this from a UX standpoint. Yes, I understand it's to bring it with parity in respect to web, but it kind of breaks user expectations (it has a hover state and hover *and click* sounds, but clicking it doesn't actually do anything).

**Review by bdach — COMMENTED** [review]
I suppose this is mostly ok?

File: osu.Game/Overlays/Profile/Sections/ProfileItemBackground.cs (hunk 1 above)
Comment (EVAST9919) [reply osu.Game/Overlays/Profile/Sections/ProfileItemBackground.cs]: Valid point. Removing sounds might work, but I'll wait for ppy's response on this.

Comment (EVAST9919) [reply osu.Game/Overlays/Profile/Sections/ProfileItemBackground.cs]: Actually, yes. In the current implementation there are no sounds, so I'll change that

**Comment by recapitalverb** [comment]
The pp display shown in the screenshot seems to be pushed too much to the right.

**Review by smoogipoo — APPROVED** [review]
Lgtm, just some minor styling fixes.

**Pull request merged**